src/page/AddMenu/components/GoMyShop/GoMyShopPopup.tsx

```diff
@@ -0,0 +1,30 @@
+import { createPortal } from 'react-dom';
+import { Link } from 'react-router-dom';
+import styles from './GoMyShopPopup.module.scss';
+
+interface CheckPopupProps {
+  isOpen: boolean;
+}
+
+export default function GoMyShopPopup({ isOpen }: CheckPopupProps) {
+  if (!isOpen) return null;
+  return createPortal(
+    <div className={styles.popup}>
+      <div className={styles.content}>
+        <span className={styles['content__main-text']}>가게 정보를 저장하시겠습니까?</span>
+        <span className={styles['content__sub-text']}>메뉴 관리에서 기존 메뉴의 정보 수정이 가능합니다.</span>
```

Comment:
제가 피그마에서 본 문구하고 다릅니다! 
확인 부탁드립니다!
```suggestion
        <span className={styles['content__main-text']}>신규 메뉴 추가 완료 되었습니다.</span>
        <span className={styles['content__sub-text']}>메뉴 관리에서 기존 메뉴의 정보 수정이 가능합니다.</span>
```

Reply:
수정완료했습니다. b84e60a2c4a4dffd82975339d2f64827149fbeef